src/docs/product/integrations/compliance/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Compliance
+sidebar_order: 95
+description: "Learn more about Sentry's compliance integrations."
+redirect_from:
```

Comment:
Try removing the `redirect_from` here, it's not needed and I think causing the issues you were talking about.

Reply:
Thanks! Appreciate you taking a look!